### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by GregoryWilusz**
Implemented Redux for log in/sign up forms + APK file.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
 import React, {Component} from 'react';
-import {Container, Header, Title, Content, Footer, FooterTab, Button, Left, Right, Icon, Text} from 'native-base';
-import {Provider} from 'react-redux';
-import {createStore} from 'redux';
+import { Container, Header, Title, Content, Footer, FooterTab, Button, Left, Right, Icon, Text } from 'native-base';
```
Comment (piotrkorlaga) [inline App.js]: lets configure eslint to keep fromatting consistent

File: App.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-import {LoginForm} from './src/LoginForm';
+import LoginForm from './src/LoginForm';
 import {SpinnerComponent} from './src/components/Spinner';
 import {RouterComponent} from "./src/Router";
 import reducers from './src/reducers';
 
 export default class App extends Component {
 
-    state = {loggedIn: null};
+    // state = {loggedIn: null};
```
Comment (piotrkorlaga) [inline App.js]: dont use comments to disable code, pls

File: App.js [hunk 3]
```diff
@@ -22,48 +23,50 @@ export default class App extends Component {
...
+    //         case false:
+    //             return <LoginForm/>;
+    //         default:
+    //             return (
+    //                 <Container>
+    //                     <SpinnerComponent/>
+    //
+    //                 </Container>
+    //             );
+    //     }
```
Comment (piotrkorlaga) [inline App.js]: dont comment code!

File: package.json [hunk 4]
```diff
@@ -11,9 +11,11 @@
 		"native-base": "^2.3.3",
 		"react": "16.0.0",
 		"react-native": "0.50.3",
+		"react-native-firebase": "^3.1.1",
```
Comment (piotrkorlaga) [inline package.json]: i think we dont use react-native-firebase so should be removed

File: src/Router.js [hunk 5]
```diff
@@ -1,18 +1,21 @@
...
     return(
         // sceneStyle will be applied to all different scenes of our App
         <Router sceneStyle={{ paddingTop: 65 }}>
 
-                <Scene key="login" component={LoginForm} title="Please Login"/>
+                <Scene key="login" component={LoginForm} title="Log in to your account"/>
+                <Scene key="signup" component={SignUpForm} title="Sign up for free"/>
 
-                <Scene key="today" component={TodayScreen} title="Step 1/3" onRight={() => Actions.tomorrow()} rightTitle="Next" />
+                <Scene key="today" component={TodayScreen} title="Step 1/3" onBack={() => {firebase.auth().signOut(); Actions.login(); console.log('Signed out')}} onRight={() => Actions.tomorrow()} rightTitle="Next" />
```
Comment (piotrkorlaga) [inline src/Router.js]: this should be move to separate method. Moreover move it to separate log out button, pls

File: App.js (hunk 1 above)
Comment (GregoryWilusz) [reply App.js]: Done

File: App.js (hunk 3 above)
Comment (GregoryWilusz) [reply App.js]: Done

File: package.json (hunk 4 above)
Comment (GregoryWilusz) [reply package.json]: Done

File: src/Router.js (hunk 5 above)
Comment (GregoryWilusz) [reply src/Router.js]: Deleted due to rearranging app's screens.

File: App.js (hunk 1 above)
Comment (piotrkorlaga) [reply App.js]: did you push?

**Pull request merged**